### PR TITLE
Creating e2e test for udp logs

### PR DESCRIPTION
**Pull request by carlosroman**
What does this PR do?
---------------------

This updates the test logger app to log in various different ways.

Which scenarios this will impact?
-------------------

None as `ghcr.io/datadog/apps-logger:main` not used anywhere yet.

Motivation
----------

Want ability to send logs to Agent via `stdout`, `stderr`, `UDP` and `TCP`.

Additional Notes
----------------

### CODE REVIEW

File: components/datadog/apps/logger/images/logger/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#  Logger app
```
Comment (pducolin) [inline components/datadog/apps/logger/images/logger/README.md:1]: **👏 praise**
Thank you for adding a doc file!

File: components/datadog/apps/logger/images/logger/main_test.go [hunk 2]
```diff
@@ -0,0 +1,222 @@
+package main
```
Comment (pducolin) [inline components/datadog/apps/logger/images/logger/main_test.go:1]: **❓ question**
Are you interested in running this on the CI? I can help! We have an integration test stage, we can add an unit test stage

Can also help migrating it to `testify`

**Review by pducolin — APPROVED** [review]
Super nice, with tests and docs 💯

File: components/datadog/apps/logger/images/logger/main_test.go (hunk 2 above)
Comment (carlosroman) [reply components/datadog/apps/logger/images/logger/main_test.go:1]: The app pretty simple and just there for some sanity checking. If it gets more complicated I'll definitely move to using `testify`.

> We have an integration test stage, we can add an unit test stage

**Pull request merged**